src/icons/two-toned/IconSharedLinkVars.scss

```diff
@@ -0,0 +1,9 @@
+@mixin icon-shared-link-style-override($primaryColor, $secondaryColor) {
```

Comment:
Is this mixin intended to be imported by the parent app? If so, can we rename it to be more generic/reusable?

Reply:
It is intended for the parent app to use. The idea @booc0mtaco had was that since we're exposing the internals of the SVG through the CSS, we could mitigate that by also providing a mixin per icon. So the parent app would style it using this mixin rather than directly styling with selectors.

Reply:
Two changes for consideration: 

- rename this file so it's a [partial import](https://sass-lang.com/guide#Partials). That way we can distinguish it from an actual SASS selectors needed for future icons/components
- we don't have a convention around naming for this situation yet. I'd lean toward something like `ComponentName.XXX.scss` replacing the XXX with `variables`, `mixins`, `custom`, or some words that denote what the file is for. This would fit with how we handle media query files, and some rising consensus about naming that have happened recently

Reply:
Done